sdk/monitor/opentelemetry-exporter-azure-monitor/test/unit/export/trace.test.ts

```diff
@@ -52,7 +52,9 @@ describe("#AzureMonitorBaseExporter", () => {
 
   describe("Sender/Persister Controller", () => {
     describe("#exportEnvelopes()", () => {
-      const scope = nock(DEFAULT_BREEZE_ENDPOINT).post("/v2/track");
+      const scope = nock(DEFAULT_BREEZE_ENDPOINT)
+        .log(console.log)
```

Comment:
This `log` in here for debugging only? Should it be removed before we commit?

Reply:
Not needed, removing it